src/App.js

```diff
@@ -1,56 +1,94 @@
 //import logo from './logo.svg';
-import logo from './imgs/mugiwara.jpg';
-import './App.css';
-import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
-import List from './containers/productList/ProductList';
-import BannerSlider from './containers/Banner/BannerSlider';
-import CategoriesSlider from './containers/CategoriesSlider/CategoriesSlider';
-import Footer from './containers/Banner/Footer';
+//import logo from './imgs/mugiwara.jpg';
+import "./App.css";
+import { useFeaturedBanners } from "./utils/hooks/useFeaturedBanners";
+//import List from './containers/productList/ProductList';
+import BannerSlider from "./containers/Banner/BannerSlider";
+import CategoriesSlider from "./containers/CategoriesSlider/CategoriesSlider";
+import Footer from "./containers/Banner/Footer";
 import Banners from "./mocks/en-us/featured-banners.json";
-import Header from './containers/Header/Header';
+import Header from "./containers/Header/Header";
+//import ProductListPage from './containers/ProductListPage/ProductListPage';
+import PageSelector from "./containers/PageSelector/PageSelector";
+import React, { useState, useEffect } from "react";
+import Categories from "./mocks/en-us/product-categories.json";
+let pages = ["Home", "PriductListPage"];
+
+
+const Home=()=>{
+return(<div><BannerSlider banners={Banners}></BannerSlider>
+  <CategoriesSlider /></div>)
+}
+
+var Cats=[];
+function getCategories(){
+  Cats=[];
+  for(let i in Categories.results){
+    let cat = Categories.results[i];
+
+    Cats.push({
+      name: cat.data.name,
+      id: cat.id,
+      selected: false
+    });
+    console.log(Cats[i].name,Cats[i]);
+  }
+}
 
 function App() {
+  getCategories();
   const { data, isLoading } = useFeaturedBanners();
   console.log(data, isLoading);
+  let [pageIndex, setPageIndex] = useState(0);
+
+  const changePage = (indx) => {
+    if(pageIndex!==indx){
```

Comment:
This verification is not necessary, React will know when you are rendering the same ui and will not rerender